lib/client/index.js

```diff
@@ -1,9 +1,10 @@
 const socket = require('./socket');
 const relay = require('../relay');
 const logger = require('../log');
+const version = require('../../package.json').version || 'local';
```

Comment:
Can we wrap this up into `lib/version.js` please, then reference it appropriately

Reply:
Will do